Kiosk/Admin/AdminLogViewController.swift

```diff
@@ -5,9 +5,9 @@ class AdminLogViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         textView.text = try? NSString(
-            contentsOf: logPath(),
-            encoding: String.Encoding.ascii.rawValue
-        ) as String
+                contentsOf: logPath(),
+                encoding: String.Encoding.ascii.rawValue
+            ) as String
```

Comment:
This is indenting twice, which night not be what you're looking for

```swift
        textView.text = try? NSString(
                contentsOf: logPath(),
                encoding: String.Encoding.ascii.rawValue
            ) as String

```

Reply:
Good eyes! This extra indentation is triggered by the `try`. It's actually something that I wanted to experiment with and apparently committed accidentally (or subconsciously?).

But it's certainly something that no one does and the extra indent is probably more confusing than helping.